assets/js/interactivity/directives.js

```diff
@@ -100,9 +100,18 @@ export default () => {
 
 	// data-wc-on--[event]
 	directive( 'on', ( { directives: { on }, element, evaluate } ) => {
+		const events = new Map();
```

Comment:
These are the fixes to allow multiple event handlers, they were lost in the recent update of the inetractivity store API.